src/order/rasiowa_sikorski.lean

```diff
@@ -0,0 +1,115 @@
+/-
+Copyright (c) 2020 David Wärn. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: David Wärn
+-/
+import order.basic
+import data.equiv.encodable
+
+/-!
+# The Rasiowa–Sikorski lemma
+
+This file contains the (dual) Rasiowa–Sikorski lemma.
+
+## Main definitions
+
+We work with a preorder `P`, a term `p : P`, and a countable family `𝒟` of cofinal subsets of `P`.
```

Comment:
Please make this docstring readable for people who don't know what is the Rasiowa-Sikorski lemma.

Reply:
I made an attempt -- what do you think?